hmcts/bulk-scan-orchestrator
# Added functionality to spin up local docker instance

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RPE-699

### Change description ###

- Added functionality to spin up CCD stack with all dependent services locally.

- Added scripts to import CCD Definitions and utilities related to Idam to create roles, users  etc.

- Tested locally

- TODO

1. Update README.

2. Add queue emulator and publish docker image for bulk scanning orchestrator to hmcts registry.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```
